test/support/wait-for.ts

```diff
@@ -0,0 +1,7 @@
+import { TIME_EXCEEDING_ACBS_AUTHENTICATION_ID_TOKEN_CACHE_TTL } from './environment-variables';
+
+// TODO APIM-97: should we use fake timers instead?
```

Comment:
In the current implementation, the tests wait in real time if they need to wait for the cache to expire. We could use something like `useFakeTimers` from `sinon` to manually increment the time but I don't think it's worth the extra complexity at this point since not many tests have to wait for the cache to expire. Let me know if you disagree!

Reply:
Looks like wait is 100 or 500ms, don't see issue with this for now.